forge-gui/res/languages/de-DE.properties

```diff
@@ -3297,3 +3297,4 @@ lblWurm=Wurm
 lblYeti=Yeti
 lblZombie=Zombie
 lblZubera=Zubera
+lblWouldYouLikeToPlaneswalk=Would you like to Planeswalk ?
```

Comment:
There should be an area for Effect prompts... the Effects are organized alphabetically. You could look for PlayEffect and put the stuff for Planeswalk effect just above